pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+var getpvlist = `
+kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
+
+func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace string) []string {
+
+	getpv := fmt.Sprintf(getpvlist)
+	// Get all csi reference on k8s side for all cephfs pv
+	k8sRef := exec.ExecuteBashCommand(getpv)
+	if k8sRef == "" {
+		logging.Info("No k8s pvs found. Nothing to do.")
+	}
+	k8sRefl := strings.Split(k8sRef, " ")
+
+	fsArgs := []string{
+		fmt.Sprintf("fs ls --format=json | jq --raw-output '.[].name'"),
+	}
```

Comment:
```suggestion
	fsArgs := []string{
		fmt.Sprintf("fs ls --format=json | jq --raw-output '.[].name'"),
	}
```

This is incorrect, do not use `fmt.Sprintf()` in `[]string{}` as it will treat it as single string in bash. To achieve the desired output
```
fsArgs := []string{
		"fs", "ls", "--format", "json",
	}
```

get output of above and use json UnMarshal to get the `name`